backlog-set-due-date

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+
+redmine_api_key="${redmine_api_key:?"Need redmine API key"}"
+project_id="${project_id:-18}"
+ticket_limit="${ticket_limit:-200}"
+host="${host:-"https://progress.opensuse.org"}"
+issues=$(mktemp)
+issue=$(mktemp)
+
+curl -s -H "X-Redmine-API-Key: $redmine_api_key" "$host/issues.json?project_id=$project_id&limit=$ticket_limit" | jq -r '.issues | .[] |  select(.priority.name!="Low") | select(.due_date==null)' >"$issues"
+
+for id in $(jq .id "$issues"); do
+  jq "select(.id==$id)" "$issues" >"$issue"
+  case $(jq ".priority.name" "$issue"| tr -d '"') in
+    Normal) days=356;;
+    High) days=30;;
+    Urgent) days=7;;
+    Inmediate) days=1;;
+    *) days=0;;
+  esac
+
+  start_date=$(jq ".created_on" "$issue" | tr -d '"')
+  due_date=$(date -d"$start_date + $days days" +%Y-%m-%dT%TZ)
+  echo "$due_date"
+  #TODO Setup the due date using the Progress API
```

Comment:
I think this needs to be changed a bit. I am trying to understand myself how "start_date" is set in redmine. E.g. in https://progress.opensuse.org/issues/73468.json the ticket was created 2020-11-11, start_date is 2020-11-15, at the time I set the assignee to myself, so there it would work. But in https://progress.opensuse.org/issues/73468.json ticket was created 2020-10-17 with no assignee but the start date is also 2020-10-17. I don't understand it, do you understand that?
The most simple approach I could think of: As soon as a ticket has an assignee set, set due-date to $now + 14 days (our median cycle time), regardless of priority. Keep it plain and simple for a start :) We can always extend it later.